crates/aptos-dkg/src/sigma_protocol/homomorphism/tuple.rs

```diff
@@ -37,6 +37,28 @@ where
     pub hom2: H2,
 }
 
+// For serde the following impl would take one line: https://serde.rs/container-attrs.html#bound
+// Alternatively, could just add CanonicalSerialize to `homomorphism::Trait`
```

Comment:
Not sure I understand the alternative & its implications.

This seems to automatically implement serialization for tuples, right?

Reply:
TODO: do the simpler alternative!